app/src/common/shared/com/igalia/wolvic/VRBrowserActivity.java

```diff
@@ -914,6 +914,20 @@ public void onTrimMemory(int level) {
         }
     }
 
+    public void exitApp() {
```

Comment:
Maybe enough with private visibility?

Also I think we should look for a more descriptive name. Something like `ShowExitAppDialog` for example.

Reply:
I changed the function name to "quiteApp", btw. Regarding your suggestion, I don't think the relevant info that the function name should provide is that it shows a dialog; what it actually does is "quitting from the app", don't you  think so ? The fact that it shows a confirmation dialog is anecdotal, IMHO.  It's similar to the WindowWidget::startDownload function, where we also show a confirmation dialog before initiating the actual operation.

Reply:
Well my personal interpretation is that the method just shows a dialog. The handlers for the buttons do other things, like exiting the app. But if you click CANCEL the function just does nothing but showing the dialog. IMO a `exitApp()` method is a method that unconditionally exits the app, and that's what I would expect without checking the actual implementation.